clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -330,8 +330,9 @@ void run(long now) {
             }
         }
 
-        long pollTimeout = sendProducerData(now);
-        client.poll(pollTimeout, now);
+        long currentTimeMs = time.milliseconds();
```

Comment:
Should this also be `now` instead of `currentTimeMs` ?

Reply:
I actually favor `currentTimeMs`, but perhaps it is unnecessarily pedantic.

Reply:
I personally don't care too much. Was just wondering about keeping the code consistent. If `now` is the current way, we might want to keep it this way -- or rename it everywhere (but maybe in a separate PR). Just my 2 cents. Feel free to ignore.